bot/exts/fun/yodaify.py

```diff
@@ -0,0 +1,89 @@
+import re
+from discord.ext import commands
+from discord import AllowedMentions
+
+from bot.bot import Bot
+
+
+class Yodaify(commands.Cog):
+    """Cog for the yodaify command."""
+
+    def _yodaify_sentence(self, sentence: str) -> str:
+        """Convert a single sentence to Yoda speech pattern."""
+        sentence = sentence.strip().rstrip('.')
+        
+        # Basic pattern matching for subject-verb-object
+        # Looking for patterns like "I am driving a car" -> "Driving a car, I am"
+        words = sentence.split()
+        if len(words) < 3: 
+            return sentence + "."
```

Comment:
Is there a reason this doesn't `return None`, as because it doesn't sentences with less than 3 words don't get the "Yodafication this doesn't need, wookie!" message even though they weren't changed.

I think you could also just remove the check, it doesn't seem like it would break anything lower down.